allure-report-face/src/components/attachment/AttachmentView.js

```diff
@@ -39,6 +39,8 @@ class AttachmentView extends ItemView {
         return $.ajax(this.sourceUrl, {dataType: 'text'}).then((responseText) => {
             if(this.type === 'csv') {
                 this.content = d3.csv.parseRows(responseText);
+            } else if(this.type === 'tab-separated-values') {
+                this.content = d3.tsv.parseRows(responseText);
```

Comment:
it would be better not to create a special flag `isTable`

```
if(this.type === 'csv' || this.type= 'tsv') {
   this.isTable = true;
   this.content = (this.type === 'csv' ? d3.csv : d3.tsv).parseRows(responseText)
}
```